.github/workflows/cpp_lint.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Lint C++ files
+
+name: cpplint
+on:
+  push:
+    paths:
+      - '**.cc'
+      - '**.h'
+      - '**.cpp'
+  pull_request:
+    paths:
+      - '**.cc'
+      - '**.h'
+      - '**.cpp'
+jobs:
+  cpplint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+    - run: pip install cpplint
+    - run: cpplint --recursive --exclude=third_party/* .
```

Comment:
Should we remove the third_party exclusion in favor of the system header?

Reply:
+1